test1.js

```diff
@@ -3,3 +3,9 @@ var foo = '' ;
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+var okey = '' ;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26958)
```suggestion
var okey = "" ;
```